### ISSUE TITLE
Adjust the bubble sizes dynamically within the scope of the map 

### ISSUE THREAD

**Issue by sasrsc** [sassoftware/sas-visualanalytics-geowebmap#5]
As you zoom in/out etc... it would be nice to be able to adjust the bubble sizes based on the measure values for the places visible within the map at that point. For example: I have a high majority of room nights in Raleigh. If I'm looking at San Diego where I am a smaller amount of nights but some hotels still have a fair amount but they just are much smaller than Raleigh then the bubbles are all small so it's difficult to discern the sizing differences. The work around is to filter the data but that's an extra step.... just saying...

**Comment by tograh** [comment]
We'll discuss this as a toggled mode.  Two alternatives I should mention.  (1) Since the range of the data determines the relative bounds of the sizes--regardless of zoom factor--you can filter directly from VA's filter panel to achieve this effect--at least until you pan manually, at which point the bounds are determined by user actions, not data range.  (We could add something to allow the user to reset that behavior after manual pans.)  (2) Another option, which would unfortunately not spare a step, would be to use the "smart legends" (useSmartLegends=true), which allow you to adjust sizes at any time.

**Comment by sasrsc** [comment]
Yes, in regards to #1 - yes, I have a filter which helps but as you say once you pan manually it's determined by the user. I like the idea of a reset on the map menu easily accessible. #2 I'll experiment with the smartLegends just to see how that feels to a user. In this particular case it's probably not relevant as I'm coloring off a binary value.
